kuma/javascript/src/payments/pages/management.jsx

```diff
@@ -0,0 +1,200 @@
+            subscription.next_payment_at
+        );
+        const lastActiveDate = formatDate(
+            locale,
+            date.setDate(date.getDate() - 1)
+        );
+        return (
+            <>
+                <p>
+                    {interpolate(
+                        'Next payment occurs on %(nextPaymentDate)s.',
+                        {
+                            nextPaymentDate,
+                        }
+                    )}
+                </p>
+                <div className="active-subscriptions">
+                    <ul>
+                        <li>
+                            <span className="label amount">
+                                {gettext('Amount')}
+                            </span>
+                            <span className="value">
+                                {/* amount is in cents, so divide by 100 to get dollars */}
+                                {formatMoney(locale, subscription.amount / 100)}
+                            </span>
+                        </li>
+                        <li>
+                            <span className="label credit-card">
+                                {gettext('Card number')}
+                            </span>
+                            <span className="value">{`**** **** **** ${subscription.last4}`}</span>
+                        </li>
+                        <li>
+                            <span className="label credit-card">
+                                {gettext('Expiry')}
+                            </span>
+                            <span className="value">
+                                {subscription.expires_at}
+                            </span>
+                        </li>
+                    </ul>
+                </div>
+                <button
+                    className="cta cancel"
+                    onClick={handleClick}
+                    type="button"
+                >
+                    {gettext('Cancel subscription')}
+                </button>
+                {showForm && (
+                    <CancelSubscriptionForm
+                        onSuccess={handleDeleteSuccess}
+                        setShowForm={setShowForm}
+                        date={lastActiveDate}
+                    />
+                )}
+            </>
+        );
+    };
+
+    const renderContent = () => {
+        switch (true) {
+            case !userData.isAuthenticated:
+                // User is not logged in
+                return (
+                    <Interpolated
+                        id={gettext('Please <signInLink /> to continue.')}
+                        signInLink={<SignInLink text={gettext('sign in')} />}
+                    />
+                );
+            case !userData.isSubscriber || canceled:
+                // Not a subscriber or just canceled their subscription
+                return renderInvitation();
+            case userData.isSubscriber && !!subscription:
```

Comment:
Just curious here. Could you be subscriber and not have any subscriptions?

Reply:
I don't believe so. This logic is here because while it's waiting for the request to get subscriptions to complete, it can fall through to show the "Loading" text.

Reply:
I guess what I was asking is whether this can simply be:

```suggestion
            case userData.isSubscriber:
```

Reply:
If we change it to that, then it will no longer show the "Loading..." text for a few seconds while it waits for the subscriptions call to complete, so this is what you'll see:
<img width="404" alt="Screen Shot 2020-04-29 at 11 44 02 AM" src="https://user-images.githubusercontent.com/650/80616413-df5fe380-8a0e-11ea-8a97-d895a5415ef8.png">